server/src/assets/entities/asset.entity.ts

```diff
@@ -32,5 +33,10 @@ export class Asset {
   quantity: number;
 
   @ManyToOne(() => User, (user) => user.assets)
+  @JoinColumn()
   poster: User;
+
+  @OneToMany(() => Transaction, (transaction) => transaction.asset, { eager: true })
```

Comment:
neat, but i don't think we want to get all the asset's transactions every time we get the asset
what about asset display page? we don't need all this extra data there
(that is what `eager: true` does right?)

Reply:
Yeah, I think you're right.    Looking at this:  https://stackoverflow.com/questions/68992271/meaning-of-using-eagertrue-option-in-typeorm    and the last line "eager loading means related data is automatically fetched."    

So just get rid of the eager on both sides, right?   Or did you mean delete the @OneToMany on the Asset side?   Can I have just a one-sided relation?

Reply:
i just meant to get rid of the `eager` piece